docs/core/tools/global-tools.md

```diff
@@ -126,33 +126,47 @@ The .NET Core CLI tries to add the default locations to the PATH environment var
 * On macOS, if you've installed the .NET Core SDK using *.tar.gz* files and not *.pkg*.
 * On Linux, you need to edit the shell environment file to configure the PATH.
 
-## Other CLI commands
+## Global tools available in the .NET Core SDK
```

Comment:
I don't like this section. It makes it sound like these tools are included in in the SDK by default. But they are not, they are still installed via nuget.

Reply:
What is this section about? What is Other CLI commands?
`aspnet-codegenerator` is included with the SDK.

Reply:
I've asked @wli3 to review. I believe these changes are misleading, so please wait until we confirm.